### PR TITLE
feat: add process component base on OGC process

### PR DESCRIPTION
**Pull request by sebbousquet**

### CODE REVIEW

File: projects/arlas-toolkit/src/lib/components/process/process.component.html [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+        <button mat-flat-button mat-dialog-close="true">{{'Cancel' | translate}}</button>
+        <button [color]="'primary'" matStepperNext mat-flat-button [disabled]="!formGroup.valid">
+          {{'Next' | translate}}</button>
+      </div>
+    </div>
+  </mat-step>
+  <mat-step label="{{'Summary' | translate}}" class="step" state="summary">
+    <div class="step">
+      <div class="summary">
+        <div class="infos">Vous êtes sur le point de lancer le téléchargement de {{nbProducts}} produits <br />avec les
```
Comment (QuCMGisaia) [inline projects/arlas-toolkit/src/lib/components/process/process.component.html:80]: There is no translate pipe here

**Review by QuCMGisaia — APPROVED** [review]

File: projects/arlas-toolkit/src/lib/components/process/process.component.html (hunk 1 above)
Comment (sebbousquet) [reply projects/arlas-toolkit/src/lib/components/process/process.component.html]: fixed

File: projects/arlas-toolkit/assets/i18n/fr.json [hunk 2]
```diff
@@ -1,14 +1,13 @@
 {
-	"Click to remove all filters of this tab": "Supprimer tous les filtres de cette onglet",
 	"Click to remove filter": "Supprimer le filtre",
+	"Click to remove all filters of this tab": "Supprimer tous les filtres de cette onglet",
```
Comment (QuCMGisaia) [inline projects/arlas-toolkit/assets/i18n/fr.json:3]: de _cet_ onglet

File: projects/arlas-toolkit/assets/i18n/fr.json [hunk 3]
```diff
@@ -93,6 +99,9 @@
 	"Confirm password": "Confirmation",
 	"Your password has been changed.": "Votre mot de passe a été modifié.",
 	"Change": "Changer",
+	"No search": "Deux caractères ou plus sont nécessaires.",
+	"Searching": "Recherche en cours...",
+	"No results": "Aucun résultats pour cette recherche.",
```
Comment (QuCMGisaia) [inline projects/arlas-toolkit/assets/i18n/fr.json:104]: Aucun _résultat_

File: projects/arlas-toolkit/assets/i18n/fr.json [hunk 4]
```diff
@@ -165,6 +178,8 @@
 	"LAST_PAGE_LABEL": "Dernière page",
 	"RANGE_PAGE_LABEL_1": "0 sur {{length}}",
 	"RANGE_PAGE_LABEL_2": "{{startIndex}} - {{endIndex}} sur {{length}}",
+	"with the follwing parameters": "avec les paramètres suivants",
+	"Toggle datasets": "Masquer datasets",
```
Comment (QuCMGisaia) [inline projects/arlas-toolkit/assets/i18n/fr.json:182]: Masquer _les_ datasets

**Review by QuCMGisaia — APPROVED** [review]

**Pull request merged**